src/main/java/io/kamax/mxisd/controller/auth/v1/io/LoginRequestJson.java

```diff
@@ -0,0 +1,105 @@
+package io.kamax.mxisd.controller.auth.v1.io;
+
+public class LoginRequestJson {
+
+    private String type;
+    private String user;
+    private String medium;
+    private String address;
+    private String password;
+    private String token;
+    private String device_id;
```

Comment:
Gson allows to write fields with Java naming convention, like `deviceId`. See GsonUtil for getting a Gson instance with the right settings.

Reply:
I wasn't aware of that.
However, from my point of view, I'll prefer to keep this naming, because this class is just a pojo reflecting the spec, I think it is easier to read and make link with the actual spec. (in one sense, the "toString" method reflects exactly the Json data)